File: CS50X/Pset1/mario-more/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Enter height pyramid
    int h;
    do
    {
        h = get_int("Enter heigt: ");
    }
    while (h <= 0 || h >= 9);
    //print pyramid
    for (int i = 1; i <= h; i++)
    {
        //print space
        for (int j = i; j < h; j++)
        {
            printf(" ");
        }
        //print first pyramid
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        //print second pyramid
        printf("  ");
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: CS50X/Pset6/sentimental-mario-more/mario.py
# TODO
import cs50

h = cs50.get_int("Height: ")
while h < 1 or h > 8:
    h = int(input("Height: "))

for i in range(h):
    print((h - i - 1) * " " + (i + 1) * "#" + "  ", end="")
    print((i + 1) * "#")
